data/it.mijorus.whisper.desktop.in

```diff
@@ -5,4 +5,5 @@ Icon=it.mijorus.whisper
 Terminal=false
 Type=Application
 Categories=Utility;GTK;
+Keywords=audio;micrphone;sound;voice;
```

Comment:
First, you have typo in here: `micrphone` instead of `microphone`
Second, I would add even more keywords:
```.desktop
Keywords=audio;voice;microphone;sound;recording;connection;speaker;
```

Reply:
Oops, you're right, and yes, those look like _sound_ suggestions, will add them in! Thanks